### PR TITLE
add failure toasts if excpetion before tx is sent

### PR DESCRIPTION
**Pull request by bthaile**

### CODE REVIEW

**Review by andrewdamelio — APPROVED** [review]

File: packages/simplified/src/modules/common/tables.tsx [hunk 1]
```diff
@@ -220,6 +220,16 @@ export const PositionFooter = ({
           .catch((error) => {
             setPendingClaim(false);
             console.error("Error when trying to claim winnings: ", error?.message);
+            addTransaction({
+              has: 'claim-failed',
```
Comment (JohnDanz) [inline packages/simplified/src/modules/common/tables.tsx]: typo

File: packages/simplified/src/modules/common/tables.tsx [hunk 2]
```diff
@@ -254,6 +264,16 @@ export const PositionFooter = ({
       .catch((error) => {
         setPendingCashOut(false);
         console.error("Error when trying to claim winnings: ", error?.message);
+        addTransaction({
+          hash: 'cash-out-failed',
+          chainId: loginAccount?.chainId,
+          seen: false,
+          status: TX_STATUS.PENDING,
```
Comment (JohnDanz) [inline packages/simplified/src/modules/common/tables.tsx]: status should be failed.

File: packages/simplified/src/modules/modal/modal-add-liquidity.tsx [hunk 3]
```diff
@@ -250,6 +260,16 @@ const ModalAddLiquidity = ({ market, liquidityModalType, currency }: ModalAddLiq
         })
         .catch((error) => {
           console.log("Error when trying to add AMM liquidity: ", error?.message);
+          addTransaction({
+            hash: 'add-liquidity-failed',
+            chainId: loginAccount.chainId,
+            from: account,
+            seen: false,
+            status: TX_STATUS.PENDING,
```
Comment (JohnDanz) [inline packages/simplified/src/modules/modal/modal-add-liquidity.tsx]: should be failure

File: packages/simplified/src/modules/modal/modal-add-liquidity.tsx [hunk 4]
```diff
@@ -224,6 +224,16 @@ const ModalAddLiquidity = ({ market, liquidityModalType, currency }: ModalAddLiq
         })
         .catch((error) => {
           console.log("Error when trying to remove AMM liquidity: ", error?.message);
+          addTransaction({
+            hash: 'remove-liquidity-failed',
+            chainId: loginAccount.chainId,
+            seen: false,
+            status: TX_STATUS.PENDING,
```
Comment (JohnDanz) [inline packages/simplified/src/modules/modal/modal-add-liquidity.tsx]: should be failure

File: packages/simplified/src/modules/portfolio/portfolio-view.tsx [hunk 5]
```diff
@@ -91,6 +91,16 @@ const handleClaimAll = (
       .catch(error => {
         setPendingClaim(false);
         console.log('Error when trying to claim winnings: ', error?.message);
+        addTransaction({
+          hash: 'claim-all-failed',
+          chainId,
+          seen: false,
+          status: TX_STATUS.PENDING,
```
Comment (JohnDanz) [inline packages/simplified/src/modules/portfolio/portfolio-view.tsx]: failure...

**Review by JohnDanz — CHANGES_REQUESTED** [review]

**Review by JohnDanz — APPROVED** [review]
lgtm

**Pull request merged**